packages/now-cli/src/commands/deploy/latest.js

```diff
@@ -330,6 +309,18 @@ export default async function main(
         meta
     };
 
+    if (createArgs.target) {
+      const allowedValues = [
+        'staging',
+        'production'
+      ];
+
+      if (!allowedValues.includes(createArgs.target)) {
+        error(`The specified ${param('--target')} ${code(createArgs.target)} is not valid`);
```

Comment:
This error already happens on line 328. Why did you repeat it?

Reply:
I think they handle different cases, but I'm not sure. IIRC this logic is copied from original code